modules/apps/segments/segments-web/test/js/components/segment_edit/SegmentEdit.es.js

```diff
@@ -196,8 +200,7 @@ describe('SegmentEdit', () => {
 
 			fireEvent.click(cancelButton);
 
-			expect(mockNavigate).toHaveBeenCalledTimes(0);
-
+			expect(navigate).toHaveBeenCalledTimes(1);
```

Comment:
The old value here was `0`, so it is a bit odd to change it to `1`. I suspect it is because you aren't resetting the mock between tests. You'll need to add an afterEach and clear the mock each time

Reply:
Aaaah, that could be it! Gonna try it out, even though just looking at the code it would make sense that its 1, but I wasn't sure so thanks for the double-check

Reply:
The problem with having `1` is that the test is no longer isolated and it becomes dependent on the previous test running. So if you ever just ran this test alone or if the other test was removed for some reason, this test would fail. Ideally each unit test should be entirely decoupled from others and be treated as it's own "unit"